src/Service/ReportImporter.php

```diff
@@ -29,70 +23,60 @@ public function import($src)
-            $report->setSysDataSentTo($this->sanitizeText($properties->HvorOverføresDataTil));
-            $report->setSysDataComeFrom($this->sanitizeText($properties->HvorKommerDataFra));
-            $report->setSysDataLocation($this->sanitizeText($properties->PlaceringAfData));
-            $report->setSysLatestDeletionDate($this->sanitizeText($properties->HvornårSlettesOplysningerneSenest));
-            $report->setSysDataProcessors($this->sanitizeText($properties->Databehandler));
-            $report->setSysDataProcessingAgreement($this->sanitizeText($properties->DatabehandleraftaleFortrolighedsaftaleValue));
-            $report->setSysDataProcessingAgreementLink($this->sanitizeText($properties->LinkTilDatabehandleraftaleFortrolighedsaftale));
-            $report->setSysAuditorStatement($this->sanitizeText($properties->RevisorerklæringTilsynValue));
-            $report->setSysAuditorStatementLink($this->sanitizeText($properties->LinkTilRevisorerklæring));
-            $report->setSysUsage($this->sanitizeText($properties->Systembrug));
-            $report->setSysRequestForInsight($this->sanitizeText($properties->AnmodningOmIndsigt));
-            $report->setSysDateUse($this->convertDate($properties->Ibrugtagning));
-            $report->setSysStatus($this->sanitizeText($properties->StatusValue));
-            $report->setSysRemarks($this->sanitizeText($properties->Bemærkninger));
-            $report->setSysObligationToInform($this->sanitizeText($properties->Oplysningspligten));
-            $report->setSysLegalBasis($this->sanitizeText($properties->RetligtGrundlag));
-            $report->setSysConsent($this->sanitizeText($properties->SamtykkeValue));
-            $report->setSysImpactAnalysis($this->sanitizeText($properties->KonsekvensanalyseValue));
-            $report->setSysImpactAnalysisLink($this->sanitizeText($properties->LinkTilKonsekvensanalyse));
-            $report->setSysAuthorizationProcedure($this->sanitizeText($properties->Autorisationsprocedure));
-            $report->setSysVersion($this->sanitizeText($properties->Version));
-            $report->setSysInternalInformation($this->sanitizeText($properties->IndsigtInterneOplysninger));
-            $report->setSysDataWorthSaving($this->sanitizeText($properties->IndeholderSystemetBevaringsværdigeDataValue));
-            $report->setSysDataToScience($this->sanitizeText($properties->VideregivelseAfOplysningerTilForskningValue));
-
-            $sysSystemOwner = '';
-            $content = $entry->xpath('sys:link[@title="SystemejerProjektejer"]//sys:entry/sys:content');
-            if (\count($content) > 0) {
-              $systemOwner = $content[0]->children('m', TRUE)->children('d', TRUE);
-              $sysSystemOwner = (string)$systemOwner->Navn;
-            }
-            $report->setSysSystemOwner($sysSystemOwner);
+            $report->setSysId($entry->{'Id'});
+            $report->setSysInternalId($this->sanitizeText($entry->{'Id'}));
```

Comment:
Why do we sanitize here, but not in line 28? If we have to sanitize, we should do it once in a new variable, `$sysInternalId`, say, and use this throughout the code.